lib/manual_and_sections_redirecter.rb

```diff
@@ -0,0 +1,74 @@
+require "gds_api/content_store"
+
+class ManualAndSectionsRedirecter
+  def initialize(args)
+    @publishing_api ||= SpecialistPublisher.services(:publishing_api)
+    @content_store ||= GdsApi::ContentStore.new(Plek.current.find("content-store"))
```

Comment:
On closer inspection we no longer use the `@content_store` at all so we can remove it (and the `require` and the `attr_reader`) entirely - the comment still applies to `@publishing-api` though.